docs/Using Fleet/Supported-host-operating-systems.md

```diff
@@ -28,7 +28,7 @@ Fleet's agent (fleetd) generated for MacOS by `fleetctl package` does not includ
 > Ubuntu 24.04 comes with Wayland enabled by default. To use X11 instead of Wayland you can set
 > `WaylandEnable=false` in `/etc/gdm3/custom.conf` and reboot.
 
-> Fedora, CentOS 8 and 9 require a [gnome extension](https://extensions.gnome.org/extension/615/appindicator-support/) and Google Chrome for Fleet Desktop.
+> Fedora, CentOS 8 and 9 require a [gnome extension](https://extensions.gnome.org/extension/615/appindicator-support/) and Google Chrome for Fleet Desktop. Fleet Desktop is supported on CentOS 8+.
```

Comment:
Should be CentOS Stream 8+

CentOS Stream 8 != CentOS 8.
CentOS 8 was discontinued in 2021, see https://en.wikipedia.org/wiki/CentOS.

That said, I have never tested Fleet Desktop on CentOS Stream 8 or 9.

Reply:
Oh ok...

@dherder and @zayhanlon do we know of any customers/prospects who use Fleet Desktop w/ CentOS Linux? I think CentOS is used for servers mostly (not workstations)

If so, maybe we can simplify the docs by mentioning that Fleet Desktop is supported on Ubuntu and Fedora (Linux flavors we test).

Reply:
> we can simplify the docs by mentioning that Fleet Desktop is supported on Ubuntu and Fedora (Linux flavors we test).

I updated the docs to call this out.